setup.py

```diff
@@ -0,0 +1,17 @@
+import setuptools
+from inyourface.version import Version
+
+
+setuptools.setup(name='inyourface',
+                 version=Version('0.0.1').number,
+                 description='In Your Face',
+                 long_description=open('README.md').read().strip(),
+                 author='Andy Yaco-Mink',
+                 author_email='andy@yacomink.com',
```

Comment:
Welp now i'm in a hole and, taco, this is horrifying: https://www.buzzfeed.com/yacomink/crushable-buscemi?utm_term=.nkweJjjL5#.wt4QBggj9